docker-entrypoint.sh

```diff
@@ -19,6 +19,17 @@ if [[ "$APPLY_MIGRATIONS" = "1" ]]; then
 fi
 
 
-# Start server TODO run with uwsgi like tunnistamo?
-python ./manage.py runserver 0.0.0.0:8000
+if [[ "$CREATE_SUPERUSER" = "1" ]]; then
+python ./manage.py shell -c "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.filter(email='admin@example.com').delete(); User.objects.create_superuser('admin', 'admin@example.com', 'adminpass')"
```

Comment:
Missing indentation

Reply:
Let's add this as a management command instead.

Reply:
doned